Lib/fontbakery/checkrunner.py

```diff
@@ -765,6 +811,111 @@ def my_check():
+            hasattr(result_message, 'code') and result_message.code == override_target
+          ):
+      override = True
+      # Break the for loop, we had a successful override.
+      break
+
+  if override:
+    if new_status is not None:
+      result_status = new_status
+    if new_message_string  is not None:
+      # If it looks like an instance of Message we reuse the code,
+      # as it is the same condition this makes totally sense.
+      result_message = Message(result_message.code, new_message_string) \
+            if hasattr(result_message, 'code') \
+            else new_message_string
+  return override, result_status, result_message
+
+def check_log_override(check, new_id, overrides, reason=None):
+  """ Returns a new FontBakeryCheck that is decorating (wrapping) check,
+    but with overrides applied to returned statuses when they match.
+
+    The new FontBakeryCheck is always a generator check, even if the old
+    check is just a normal function that returns (instead of yields)
+    its result. Also, the new check yields an INFO Status for each
+    overridden original status.
+
+    Arguments:
+
+    check: the FontBakeryCheck to be decorated
+    new_id: string, must be unique of course and should not(!) be check.id
+            as we essentially create a new, different check.
+    overrides: a tuple of override triple-tuples
+               ((override_target, new_status, new_message_string), ...)
+               override_target: Status or string, specific Message.code
+               new_status: Status or None, keep old status
+               new_message_string: string or None, keep old message
+  """
+  @wraps(check) # defines __wrapped__
+  def override_wrapper(*args, **kwds):
+    # A check can be either a normal function that returns one Status or a
+    # generator that yields one or more. The latter will return a generator
+    # object that we can detect with types.GeneratorType.
+    result = check(*args, **kwds)  # Might raise.
+    if not isinstance(result, types.GeneratorType):
+      # Now it iterates
+      # make these always iterators, it's nicer to handle
+      # also we can mix-in new status messages
+      result = (result, )
+    # Iterate over sub-results one-by-one, list(result) would abort on
+    # encountering the first exception.
+    for (status, message) in result:  # Might raise.
+      overriden, result_status, result_message = \
+                            _check_log_override(overrides, status, message)
+      if overriden:
+        # nothing changed (despite of a match in override rules)
+        if result_status == status and result_message == message:
+          yield INFO, ('A check status override rule matched but did not '
+                      'change the resulting status.')
+        # Both changed
+        elif result_status != status and result_message != message:
+          yield INFO, ('Overridden check status and message, original:'
```

Comment:
I think it may be unnecessary to emit this INFO (it may even be confusing for the users)

Reply:
I just see that we have a `DEBUG` status. I think that's more appropriate.

Reply:
Now `DEBUG` is used and it shows only when `-l DEBUG` is specified.